src/events/rawChannelDelete.ts

```diff
@@ -0,0 +1,78 @@
+import { Snowflake } from '@klasa/snowflake';
+import { DbSet } from '@lib/structures/DbSet';
+import { WSChannelDelete } from '@lib/types/DiscordAPI';
+import { Events } from '@lib/types/Enums';
+import { GuildSettings } from '@lib/types/settings/GuildSettings';
+import { APIErrors, Time } from '@utils/constants';
+import { api } from '@utils/Models/Api';
+import { resolveOnErrorCodes } from '@utils/util';
+import { Event, EventStore } from 'klasa';
+
+export default class extends Event {
+
+	public constructor(store: EventStore, file: string[], directory: string) {
+		super(store, file, directory, { name: 'CHANNEL_DELETE', emitter: store.client.ws });
```

Comment:
Could this possibly be implemented as an Events enum entry?

Reply:
That's something we can do at a later time and is out-of-scope for this PR tbh. If you want to create a GH issue for it go ahead.